### PR TITLE
Add link

### LINKED ISSUE #65: Add link to contact
As an NUS computing student, I wish to add the links to my contacts’ socials, such as linkedin, so that I can easily contact them via social media.

### PR DESCRIPTION
**Pull request by nknguyenhc**

### CODE REVIEW

**Review by awhb — APPROVED** [review]
LGTM! Thanks!

File: src/main/java/networkbook/logic/commands/EditCommand.java [hunk 1]
```diff
@@ -148,8 +149,8 @@ public String toString() {
     public static class EditPersonDescriptor {
         private Name name;
         private Phone phone;
-        private UniqueList<Email> emails;
-        private UniqueList<Link> links;
+        private Optional<UniqueList<Email>> emails = Optional.empty();
+        private Optional<UniqueList<Link>> links = Optional.empty();
```
Comment (Singa-pirate) [inline src/main/java/networkbook/logic/commands/EditCommand.java:152]: Referring to the discussion [here](https://stackoverflow.com/questions/29033518/is-it-a-good-practice-to-use-optional-as-an-attribute-in-a-class), using `Optional` as a field type is not the recommended practice. 

This is not an urgent issue. When you are free, can you change these fields to be of type `UniqueList<>`, and use the `Optional` constructor in the getter method instead? You'll also need to change the setter methods as well, can refer to my implementation of `tags` if you want. Thanks!

**Review by Singa-pirate — COMMENTED** [review]
LGTM in general, added one request for change which is not very urgent for this release. Please consider to make the change when you are free.

File: src/main/java/networkbook/logic/commands/EditCommand.java (hunk 1 above)
Comment (awhb) [reply src/main/java/networkbook/logic/commands/EditCommand.java:152]: Noted wth thanks!

**Pull request merged**